src/modules/75-cf/pages/onboarding/views/TestYourFlagView.tsx

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useEffect, useMemo, useState, useRef } from 'react'
+import { Container, Text, Heading, Layout, Icon } from '@wings-software/uicore'
+import { useParams } from 'react-router-dom'
+import { Color } from '@harness/design-system'
+import { Classes, Switch } from '@blueprintjs/core'
+import { String, useStrings } from 'framework/strings'
+import type { PlatformEntry } from '@cf/components/LanguageSelection/LanguageSelection'
+import routes from '@common/RouteDefinitions'
+import { ApiKey, Feature, useGetAllFeatures } from 'services/cf'
+import { useToggleFeatureFlag } from '@cf/hooks/useToggleFeatureFlag'
+import { TestFlagInfoView } from './TestFlagInfoView'
+import css from './TestYourFlagView.module.scss'
+
+const POLLING_INTERVAL_IN_MS = 3000
+
+export interface TestYourFlagViewProps {
+  flagInfo: Feature
+  language: PlatformEntry
+  apiKey: ApiKey
+  environmentIdentifier: string | undefined
+  testDone: boolean
+  setTestDone: React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
Technically this is correct as it happens to be state on the other end, but you can make it easier to read and more flexible if you use something more generic
```suggestion
  setTestDone: (testDone: boolean) => void
```

Reply:
I didn't work on this file, these are Laura's recent changes pulled in after rebasing.

Reply:
Something weird must've gone wrong for these to show in the PR if they just came in from a rebase 🤔